### PR TITLE
4563 replace instance and audit logs dropdowns

### PR DESCRIPTION
**Pull request by cstns**
## Related Issue(s)

closes https://github.com/FlowFuse/flowfuse/issues/4563

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: frontend/src/pages/instance/AuditLog.vue [hunk 1]
```diff
@@ -39,8 +39,9 @@ export default {
         instance () {
             this.$refs.AuditLog?.loadEntries()
         },
-        'team.id' () {
-            this.loadUsers()
+        'team.id': {
+            handler: function (teamId) { if (teamId) this.loadUsers() },
```
Comment (cstns) [inline frontend/src/pages/instance/AuditLog.vue:43]: addressed a bug where team members were not loaded when the page loaded, resulting in an empty users list

**Review by Steve-Mcl — COMMENTED** [review]
Happy to approve @cstns but firstly, I am curious why we are only doing the Envent Type and User but not the Event Scope immediately above?

Should the items on a single page not be done in tandem?

Happy to proceed, but thought to ask in case it was an oversight.

chrome_BNVadnwdBU

**Comment by cstns** [comment]
It wasn't oversight, It's just the crazy way I managed to split the input's when creating the tasks,  Should be fixed by https://github.com/FlowFuse/flowfuse/issues/4561

**Review by Steve-Mcl — APPROVED** [review]

**Comment by Steve-Mcl** [comment]
@cstns are we merging this? It is ready to go now.

**Comment by cstns** [comment]
We will not, there's something wrong with the audit log user selector which the e2e tests picked up. Will look at it in more depth after the release

**Comment by cstns** [comment]
should be good to go after everything goes green

**Pull request merged**